lib/src/command/add.dart

```diff
@@ -415,7 +422,13 @@ Specify multiple sdk packages with descriptors.''');
     if (!packageNameRegExp.hasMatch(packageName)) {
       usageException('Not a valid package name: "$packageName"');
     }
-    final descriptor = match.namedGroup('descriptor');
+
+    // Explicitly adds a trailing space for commas and colons to help parsing
+    // a Yaml node. "key:value" will not be recognized as a valid entry.
+    final descriptor = match.namedGroup('descriptor')?.replaceAllMapped(
```

Comment:
Hmm - I'm not sure I like this. How do we know the colon is not inside a quoted yaml string?

Reply:
We don't, but we don't need to care about it IMO. Writing a completely bad syntax will fail as before.